### PR TITLE
feat(DDI-999): Added examples for dropdown props

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Added examples of leadingicon, maxheight, placeholder, width, and disabled properties for the dropdown component

### CODE REVIEW

File: libs/docs/src/components/common/Dropdown.stories.mdx [hunk 1]
```diff
@@ -359,6 +359,366 @@ export const Template = (args) => {
...
+          <GoADropdownItem value="red" label="Red" />
+          <GoADropdownItem value="green" label="Green" />
+          <GoADropdownItem value="blue" label="Blue" />
+        </GoADropdown>
+      `}
+    />
+  </Tab>
+</Tabs>
+
+#### leading Icon
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/Dropdown.stories.mdx:407]: Uppercase L

Comment (Radha-Alberta) [reply libs/docs/src/components/common/Dropdown.stories.mdx]: Changed.

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**